Program.cs

```diff
@@ -0,0 +1,410 @@
+                                break;
+                        }
+                    break;
+                }
+            }
+
+            void MenuPrincipal()
+            {
+                Console.WriteLine("1. Mantenimiento");
+                Console.WriteLine("2. Registro de Venta");
+                Console.WriteLine("3. Reportes");
+            }
+
+            void MenuMaintenance()
+            {
+                Console.WriteLine("1. Cliente");
+                Console.WriteLine("2. Producto");
+                Console.WriteLine("3. Vendedor");
+                Console.WriteLine("9. Menu Principal");
+            }
+
+            void MenuReport()
+            {
+                Console.WriteLine("1. Ventas");
+                Console.WriteLine("2. Clientes mayores 55");
+                Console.WriteLine("3. Clientes cumplan este mes");
+                Console.WriteLine("4. Productos por debajo stock");
+            }
+
+            void RegistryCustomer()
+            {
+                Console.WriteLine("Codigo Cliente: ");
+                if (!int.TryParse(Console.ReadLine(), out int idCustomer))
+                {
+                    return;
+                }
+
+                Console.WriteLine("Nombre: ");
+                string nameCustomer = Console.ReadLine();
+
+                Console.WriteLine("Fecha de Nacimiento (MM/dd/yyyy): ");
+                string birthDateCustomer = Console.ReadLine();
+
+                Customers customers = new Customers();
+                customers.IdCustomer = idCustomer;
+                customers.Name = nameCustomer;
+                customers.BirthDate = birthDateCustomer;
+
+                customersList.Add(customers);
+
+                MenuPrincipal();
+                ProcesoPrincipal();
+            }
+
+            void RegistryProduct()
+            {
+                Console.WriteLine("Codigo Producto: ");
+                if (!int.TryParse(Console.ReadLine(), out int idProduct))
+                {
+                    return;
+                }
+
+                Console.WriteLine("Descripcion: ");
+                string description = Console.ReadLine();
+
+                Console.WriteLine("Precio: ");
+                if (!decimal.TryParse(Console.ReadLine(), out decimal price))
+                {
+                    return;
+                }
+
+                Console.WriteLine("Stock: ");
+                if (!int.TryParse(Console.ReadLine(), out int stock))
+                {
+                    return;
+                }
+
+                Product product = new Product();
+                product.IdProduct = idProduct;
+                product.Description = description;
+                product.Price = price;
+                product.Stock = stock;
+
+                productList.Add(product);
+
+                MenuPrincipal();
+                ProcesoPrincipal();
+            }
+
+            void RegistrySeller()

```

Comment:
As I told you, the methods is better at the same level of all of them, not inside, just when is a delegate or maybe a lambda expresion.